recipes/r-fsa/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = '0.9.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-fsa
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/FSA_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/FSA/FSA_{{ version }}.tar.gz
+  sha256: 766196ccf3528d058822df7fbc10da731a3a6fe65cf90f1a3a1013cf99c94ca5
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-car
+    - r-dplyr
+    - r-dunn.test
+    - r-lmtest
+    - r-plotrix
+    - r-sciplot
+    - r-withr
+  run:
+    - r-base
+    - r-car
+    - r-dplyr
+    - r-dunn.test
+    - r-lmtest
+    - r-plotrix
+    - r-sciplot
+    - r-withr
+
+test:
+  commands:
+    - $R -e "library('FSA')"           # [not win]
+    - "\"%R%\" -e \"library('FSA')\""  # [win]
+
+about:
+  home: https://github.com/droglenc/FSA
+  license: GPL-2.0-or-later
+  summary: A variety of simple fish stock assessment methods. Detailed vignettes are available
+    on the fishR website <http://derekogle.com/fishR/>.
+  license_family: GPL2
+  license_file:
+    - LICENSE

```

Comment:
```suggestion
    - {{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2
```

Reply:
This uses the `r-base` installed licensed file. Please delete the LICENSE file you added in the PR.